docs/backend-api/resources/tracking/tracker/rules/rule_types.md

```diff
@@ -100,171 +112,194 @@ A rule that triggers on deviations from the route.
+| type           | Default `type`: "idling_soft".                                                      | [enum](../../../../getting-started.md#data-types) |
+| primary_text   | Primary text of rule notification when excessive idling detected by platform.       | string                                            |
+| secondary_text | Secondary text of rule notification when excessive idling end detected by platform. | string                                            |
+| param          | Idle duration to send notification.                                                 | int                                               | 
 
 ***
 
-### Auto geofencing
+### Fuel level change
 
-A rule that triggers on auto geofencing.
+Rule triggered by a drastic change in fuel level. A drastic change is when the fuel level changes faster than the 
+accuracy of the sensor in a span of ten minutes.
 
 #### parameters
 
-| name | description                      | type                                              |
-|------|----------------------------------|---------------------------------------------------|
-| type | Default `type`: "auto_geofence". | [enum](../../../../getting-started.md#data-types) |
+| name           | description                                                             | type                                              |
+|----------------|-------------------------------------------------------------------------|---------------------------------------------------|
+| type           | Default `type`: "fuel_level_leap".                                      | [enum](../../../../getting-started.md#data-types) |
+| primary_text   | Primary text of rule notification on drastically fuel level increase.   | string                                            |
+| secondary_text | Secondary text of rule notification on drastically fuel level decrease. | string                                            |
+
+#### extended parameters
+
+| name      | description           | type |
+|-----------|-----------------------|------|
+| sensor_id | ID of tracked sensor. | int  |
 
 ***
 
-### Fall detection
+### Harsh driving
 
-A rule that triggers on fall detection.
+A rule that triggers on harsh driving. Based on the configs on the device side.
 
 #### parameters
 
-| name | description                 | type                                              |
-|------|-----------------------------|---------------------------------------------------|
-| type | Default `type`: "g_sensor". | [enum](../../../../getting-started.md#data-types) |
+| name         | description                                                        | type                                              |
+|--------------|--------------------------------------------------------------------|---------------------------------------------------|
+| type         | Default `type`: "harsh_driving".                                   | [enum](../../../../getting-started.md#data-types) |
```

Comment:
Why you didn't add primary_text/secondary_text for all of rules?
But, if we will add it, we also should add other common parameters for all rules. I think it's a good point due to docs visability and readability. Because, the separating to commons and non-commons tables can lead to misunderstanding.

Reply:
changed everything in types and added primary text for all rules too. Also, moved common parameters to rules.md